src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person:
+  { name,
+    age,
+    partnerName,
+    sex,
+    isMarried } }) => (
```

Comment:
This syntax is usually hard to read. Let's destructure it inside components body